content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/ios.md

```diff
@@ -268,19 +268,46 @@ Datadog.setUserInfo(id: "1234", name: "John Doe", email: "john@doe.com")
 
 You can use the following properties in `Datadog.Configuration` when creating the Datadog configuration to initialize the library:
 
+`clientToken`
+: Either the RUM client token (which supports RUM, Logging, and APM) or regular client token, (which supports Logging and APM).
+
+`env`
+: The environment name that is sent to Datadog. This can be used to filter events on different environments (such as `staging` or `production`).
+
 `site`
-: Sets the Datadog server endpoint that data is sent to.
+: Sets the Datadog server endpoint that data is sent to. The default value is `.us1`.
+
+`service`
+: The service name associated with data sent to Datadog. The default value is set to the application bundle identifier.
+
+`bundle`
+: The bundle object that contains the current executable.
 
 `batchSize`
 : Sets the preferred size of batched data uploaded to Datadog. This value impacts the size and number of requests performed by the RUM iOS SDK (small batches mean more requests, but each request becomes smaller in size). Available values include: `.small`, `.medium`, and `.large`.
 
 `uploadFrequency`
 : Sets the preferred frequency of uploading data to Datadog. Available values include: `.frequent`, `.average`, and `.rare`.
 
+`proxyConfiguration`
+: Enter a proxy configuration attribute, which can be used to enable a custom proxy for uploading tracked data to Datadog's intake.
+
+`encryption`
+: Data encryption to use for on-disk data persistency by providing an object that complies with the `DataEncryption` protocol.
+
+`serverDateProvider`
+: A custom NTP synchronization interface. By default, the Datadog SDK synchronizes with dedicated NTP pools provided by the [NTP Pool Project][13]. Using different pools or setting a no operation `ServerDateProvider` implementation results in a de-synchronization of the SDK instance and the Datadog servers. This can lead to significant time shifts in RUM sessions or distributed traces.
```

Comment:
```suggestion
: A custom NTP synchronization interface. By default, the Datadog SDK synchronizes with dedicated NTP pools provided by the [NTP Pool Project][13]. Using different pools or setting a no operation `ServerDateProvider` implementation results in de-synchronization of the SDK instance and Datadog servers. This can lead to significant time shifts in RUM sessions or distributed traces.
```

Reply:
Would it be possible to put the last two sentences of this in a warning format?

Reply:
No, unfortunately there's no nice way to do this. The alert box breaks this table, and adding **Note**: breaks the NTP Pool Project link.